dts/bindings/stepper/trinamic,tmc5041.yaml

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+description: TRINAMIC TMC5041
+
+compatible: "trinamic,tmc5041"
```

Comment:
Should we call this `adi-trinamic`? Trinamic is a subsidiary of Analog Devices. I'm not sure what Zephyr's standard is in that situation.